src/qute/commands/registerCommands.ts

```diff
@@ -69,6 +82,40 @@ function registerJavaDefinitionCommand(context: ExtensionContext) {
   }));
 }
 
+/**
+ * Toggle for enabling/disabling Qute validation.
+ *
+ * @param context
+ */
+function registerQuteValidationToggleCommand(context: ExtensionContext) {
+  // Toggle on
+  context.subscriptions.push(commands.registerCommand(QuteSettings.QUTE_VALIDATION_ENABLED + QuteSettings.TOGGLE_ON, async (uri?: Uri) => {
+    const templateUri = uri.toString();
+    // from QuteTemplateValidationStatusCommandHandler
+    const result = await commands.executeCommand(QuteSettings.QUTE_VALIDATION_TEMPLATE_STATUS, templateUri);
+    const edit = {
```

Comment:
It's not like this how he should work.

At first I suggest that you define an interface like this:

```ts
interface TemplateValidationStatus {
	validationEnabled: boolean;
	excluded: [string];
}
```

And the code should look like this:

```ts
    const result: TemplateValidationStatus = await commands.executeCommand(QuteServerCommandConstants.QUTE_VALIDATION_TEMPLATE_STATUS, templateUri);
    if (!result.validationEnabled) {
      // The current settings 'is 'qute.validation.enabled' is set to false, update it to true
      const edit = {
        scopeUri: templateUri,
        value: true,
        editType: ConfigurationItemEditType.Update,
        section: QuteSettings.QUTE_VALIDATION_ENABLED
      } as ConfigurationItemEdit;
      await commands.executeCommand(QuteClientCommandConstants.COMMAND_CONFIGURATION_UPDATE, edit);
    }
    if (result.excluded.length > 0) {
      // The current settings 'is 'qute.validation.excluded' have some patterns which matches the template uri
      // Those patterns must be removed
      const edit = {
        scopeUri: templateUri,
        value: result.excluded,
        editType: ConfigurationItemEditType.UpdateByRemoving, // TODO : implement UpdateByRemoving which must remove the given value (excluded array) from the existing qute.validation.excluded
        section: QuteSettings.QUTE_VALIDATION_EXCLUDED
      } as ConfigurationItemEdit;
      await commands.executeCommand(QuteClientCommandConstants.COMMAND_CONFIGURATION_UPDATE, edit);
    }
    await commands.executeCommand('setContext', 'quteValidationEnabled', true);

```

 * `QUTE_VALIDATION_TEMPLATE_STATUS` is a Qute LS command server, so you define it in `QuteServerCommandConstants.QUTE_VALIDATION_TEMPLATE_STATUS` and remove `QuteSettings.QUTE_VALIDATION_TEMPLATE_STATUS`
 * you must implement `ConfigurationItemEditType.UpdateByRemoving` in order to remove the excluded patterns coming from the command which are declared in the `qute.validation.excluded` settings.json.